rasa/nlu/extractors/extractor.py

```diff
@@ -164,13 +166,28 @@ def convert_predictions_into_entities(
                 )
                 entities.append(entity)
             else:
-                entities[-1][ENTITY_ATTRIBUTE_END] = token.end
-                if confidences is not None:
-                    self._update_confidence_values(entities, confidences, idx)
+                if last_token_end == token.start - 1:
+                    entities[-1][ENTITY_ATTRIBUTE_END] = token.end
+                    if confidences is not None:
+                        self._update_confidence_values(entities, confidences, idx)
+                else:
+                    # create a new entity as the two entities are separated by a comma
+                    # or some other symbol
```

Comment:
what about spaces `" "`?

Reply:
If you have multiple spaces between an two tokens of the same entity type, we treat them as separate entities. Do you think this could be a problem?

Reply:
no that seems fine, I just wasn't sure what would happen. but your comment in the code is very clear now!